dbt/include/databricks/macros/adapters.sql

```diff
@@ -1,19 +1,11 @@
-{% macro dbt_databricks_file_format_clause() %}
+{% macro databricks__file_format_clause() %}
```

Comment:
Just curious, if a user has re-defined the `file_format_clause` in an existing project, for example
```
{% macro file_format_clause() %}
...
{% endmacro %}
```
Will this change (and the upstream change in dbt-spark) break the existing pipeline when the user upgrades the adapter version?

Reply:
According to @jtcohen6's explanation in https://github.com/databricks/dbt-databricks/pull/72#issuecomment-1086867013, the user's `file_format_clause` will be used.

> 1. Macros in the user's own project — they can always override the builtins, and they can do it by defining a macro named any of these: `file_format_clause`, `databricks__file_format_clause`, `spark__file_format_clause`, `default__file_format_clause`